lib/resources/sounds.js

```diff
@@ -20,7 +22,9 @@ const getCompanySoundsLimited = (params) => {
     endpoint: `${API_PREFIX}/sounds`,
     qs: { duration }
   }
-  return Api.unauthorizedRequest(options)
+
+  return Api.authorizedRequest(options)
+    .then((sounds) => addGetSoundByIdToSounds(sounds))
```

Comment:
you can write .then(addGetSoundByIdToSounds)
and rename `addGetSoundByIdToSounds` function to `appendSoundMethods`